packages/php-wasm/compile/php/php_wasm.c

```diff
@@ -25,6 +25,7 @@
 #include "rfc1867.h"
 #include "SAPI.h"
 #include "proc_open.h"
+#include "dns_polyfill.c"
```

Comment:
`#include`ing a `*.c` file seems a bit unorthodox. Doesn't this leave us open to having issues with double definitions down the track?

I feel it would be more typical to ensure `dns_polyfill.c` gets compiled separately and then linked into the final output? i.e. including it in the build command here:
https://github.com/WordPress/wordpress-playground/blob/35250453f431d58624287a201f1506bf0eef2b75/packages/php-wasm/compile/php/Dockerfile#L932

We'd need another header file to declare the functions so they can be included in the `additional_functions` array. But still, that seems a bit more conventional.

But I might be missing something. Was there a reason why this needed to be included as a C file? Could the DNS functions simply be added to the giant `php_wasm.c` file?

Reply:
@bgrgicak I gave this a try on a branch of my own and it seems to be working: https://github.com/p-jackson/wordpress-playground/commit/e331922e8be94fdf3b5e408586a1ca90d9dfd9aa

I'm not familiar with all of PHP's C macros, but chatgpt said the same `PHP_FUNCTION` macro can be used for declaring functions too. I also had to move the arginfo declarations into the header file since that's important info that needs to be part of the ccall to register the functions.

Reply:
Great work! Thank you for working on it. I will add these changes to the PR later today and if everything works we can merge it.

Reply:
I added the header file in a51fe85.
Thank you for helping me!